### PR TITLE
Add tests for Import-Module -Force

### PR DESCRIPTION
**Pull request by rjmholt**
## PR Summary

Add testing for the behaviour of Import-Module -Force.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [ ] Not Applicable or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1 [hunk 1]
```diff
@@ -272,3 +272,44 @@ Describe "Circular nested module test" -tags "CI" {
         $m.NestedModules[0] | Should -Be $m
     }
 }
+
+Describe "Import-Module -Force behaviour" {
```
Comment (iSazonov) [inline test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1]: Please add -Tag "CI".

**Review by iSazonov — APPROVED** [review]
LGTM with one comment.

**Comment by rjmholt** [comment]
Sorry @iSazonov, I was demonstrating to some people how to contribute simple tests to the PowerShell repo.

But I would actually like to add this test and bunch of others, so I'll add a few more to this and we can make it into a more useful PR.

**Comment by iSazonov** [comment]
We could merge this as is. You can make new PR for new tests.

**Comment by rjmholt** [comment]
@iSazonov Done!

**Comment by iSazonov** [comment]
I guess you will add tests for nested module scenarios with `-Force`.

**Comment by iSazonov** [comment]
@adityapatwardhan Could you please review?

File: test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1 [hunk 2]
```diff
@@ -272,3 +272,44 @@ Describe "Circular nested module test" -tags "CI" {
         $m.NestedModules[0] | Should -Be $m
     }
 }
+
+Describe "Import-Module -Force behaviour" -Tags "CI" {
+    BeforeAll {
+        $moduleContent = 'function Test-One { return 101 }'
+        $newModuleContent = "function Test-One { return 93 }`nfunction Test-Two { return 14 }"
+        $modulePath = Join-Path $TestDrive 'ipmofTestModule.psm1'
+        # Another edit
```
Comment (adityapatwardhan) [inline test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1]: Please remove this line.

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

**Comment by iSazonov** [comment]
@rjmholt Please rebase to reconfigure CIs

**Comment by iSazonov** [comment]
@adityapatwardhan Please update your code review.

**Review by adityapatwardhan — APPROVED** [review]

**Pull request merged**